### PR TITLE
Deploying Meals Project v1

### PR DESCRIPTION
**Pull request by Errytagedesign**
## In this branch we did:
- Find external API
- Add assets
- Setup project with webpack and jest
- Add header 
- Add footer
- Rendered list of items on to the homepage
- Add comment pop up functionality
- Render item details based on item id
- Render comments
- Add create comments functionality
- Add comment counts
- Add item counts to homepage
- Add test for the counts function using jest
- Update Readme file
- Recorded a [video](https://drive.google.com/file/d/1anXU_JlejNlw6Gb6xpVchUB_LMBdEZ0G/view?usp=sharing)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { getMeals } from './modules/getMeals';
+// import { getMealsCategory } from './modules/getMealsCategories';
+
+// Meals url
+const mealsUrl = 'https://www.themealdb.com/api/json/v1/1/search.php?f=s';
+
+window.addEventListener('load', () => {
+  getMeals(mealsUrl);
+  // getMealsCategory();
```
Comment (VICTOR-CHUKWU) [inline src/index.js:14]: - [ ] kindly remove commented codes to keep a clean code base

File: src/modules/comments.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+// involvement API
+
+import { clearInput } from './clearInput';
+import { renderComments } from './rendComments';
+
+/* eslint-disable operator-linebreak */
```
Comment (VICTOR-CHUKWU) [inline src/modules/comments.js:6]: - [ ] linters are here to help you keep a clean code base, disabling it does not help, instead try solving the error

File: src/modules/comments.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+const baseUrl =
+  'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
+/* eslint-disable operator-linebreak */
```
Comment (VICTOR-CHUKWU) [inline src/modules/comments.js:9]: - [ ] same here kindly remove linter disable and fix the error

File: src/modules/getMealsCategories.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+const getMealsCategory = async () => {
+  try {
+    const res = await fetch(
+      'https://www.themealdb.com/api/json/v1/1/lookup.php?i=52770',
+    );
+    // const res = await fetch(
+    //   'https://www.themealdb.com/api/json/v1/1/categories.php',
+    // );
+
```
Comment (VICTOR-CHUKWU) [inline src/modules/getMealsCategories.js:9]: - [ ] kindly remove commented codes to keep a clean code base

File: src/modules/renderMeals.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
+const { addLike, getAndUpdateLikes } = require('./addLike');
+const { getCounts } = require('./getCount');
+const { handleModalToggle } = require('./modal');
+const { fetchMeal } = require('./renderDetails');
+
+const home = document.querySelector('#home');
+const homeLink = document.querySelector('#homeLink');
+
+// involvement API
+/* eslint-disable operator-linebreak */
```
Comment (VICTOR-CHUKWU) [inline src/modules/renderMeals.js:10]: - [ ] linters are here to help you keep a clean code base, disabling it does not help, instead try solving the error

File: src/modules/renderMeals.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+const baseUrl =
+  'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
+/* eslint-disable operator-linebreak */
```
Comment (VICTOR-CHUKWU) [inline src/modules/renderMeals.js:13]: - [ ] linters are here to help you keep a clean code base, disabling it does not help, instead try solving the error

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @balatstar  and @Errytagedesign , 
## STATUS: CHANGES REQUIRED :recycle: 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @balatstar & @Errytagedesign 

Good job implementing the suggested changes💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Commented code is removed
-  ✅ Your README looks good, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**